labs/1.md

```diff
@@ -5,24 +5,35 @@ This hands on lab is a simple deployment of [Kubefed Operator](https://operatorh
 clusters. This lab will use the KubeFed operator above as well as the binary `kubefedctl`.
 We will deploy cluster scoped KubeFed and deploy applications to multiple clusters using KubeFed Primitives.
 
+<a id="markdown-accesing-the-clientvm" name="accesing-the-clientvm"></a>
+# Accesing the ClientVM
+All the work will be done using the ClientVM that you get from GuidGrabber for Cluster1 Environment. Let's go ahead an login into the ClientVM. Remember that login details are presented by GuidGrabber.
+
+e.g:
+
+![SSH Details GuidGrabber](assets/ssh-details-gg.png)
+
+For *sshing* in the ClientVM you can use your favorite ssh-client.
+
+~~~sh
+ssh student@clientvm.GUID.DOMAIN
+
+e.g: ssh student@clientvm.d5fe.sandbox253.opentlc.com
+~~~
+
 <a id="markdown-pre-requisites" name="pre-requisites"></a>
 # Prerequisites
-Three OpenShift clusters have already been deployed for use in this lab. During
-this lab we will utilize the `kubefedctl` binary to join the three clusters
+At this point you should be logged in ClientVM and you should have all three environment information from GuidGrabber.
+
```

Comment:
in to the ClientVM and you should have the information for all three environments defined by the GuidGrabber.